### DISCUSSION TITLE
How to change chdir

### DISCUSSION THREAD

**Discussion by volga629-1** [nvim-tree/nvim-tree.lua#2809] [Q&A] [answered]
I found Lua function 

https://github.com/nvim-tree/nvim-tree.lua/wiki/Recipes#change-root-to-global-current-working-directory

But how to use it or apply in latest version ?

Thank you

**Comment by alex-courtis** [comment, marked as answer]
You'll need to add the `vim.keymap.set` to your `on_attach`, to add it to nvim-tree mappings. See `:help nvim-tree-mappings`.
